cypress/locators/drawer/index.js

```diff
@@ -0,0 +1,9 @@
+import {
```

Comment:
We could move mentioned below locator into this file:
```suggestion
export const drawerSidebarContentInnerElement = (index) => drawerSidebarContent().find(`li:nth-child(${index})`)
```

Reply:
fixed 07d78ca